rxdogtag/src/main/java/com/uber/rxdogtag/RxDogTagTaggedExceptionReceiver.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2019. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.rxdogtag;
+
+import io.reactivex.exceptions.OnErrorNotImplementedException;
+
+/**
+ * A marker type to indicate that RxDogTag's should pass the decorated stacktrace to {@link
+ * #onError(Throwable)}. Note that this will always be tried directly, and no guarded delegate
+ * behavior will be attempted even if {@link RxDogTag.Builder#guardObserverCallbacks(boolean)} is
```

Comment:
> no guarded delegate behavior will be attempted

What's the reasoning here? Can we add why we won't do the guarded delegate behavior here?

Reply:
Because you'd be handing the modified rdt exception potentially to the guarded delegate call, likely resulting in garbled information.

Reply:
Ok gotcha. Still think it might be worth it to mentioning that in the doc.

Reply:
I don't think that's necessary, docs should be succinct and explain what something does, not why it doesn't do xyz IMO.